### PR TITLE
Instructions on overriding Appsody home dir for Win enterprise users

### LINKED ISSUE #249: Document usage of APPSODY_MOUNT_HOME to override home dir in mounts
**Is your feature request related to a problem? Please describe.**
In some circumstances, the developer might be interested in overriding the resolution of `~` during the mount phase of `appsody run` - for instance, to overcome permissions limitations on the actual home directory on Windows 10 with Enterprise AAD. The `APPSODY_MOUNT_HOME` env var can be set by the developer to achieve that purpose, but its existence and usage is not documented.

**Describe the solution you'd like**
Document the usage and effects of setting `APPSODY_MOUNT_HOME`

**Additional context**
This variable was introduced to accommodate Codewind requirements - but it can be used to work around permission issues on the actual developers' home directory. This becomes particularly useful with the Java stacks, since they attempt to mount the user's `~/.m2/repository`. On Win 10 with Enterprise AAD, the user identity under which the container runs has no access to the developer's home - and the developer has no simple way to give permissions on the home directory to locally defined users. A way around this situation is to let the Appsody container mount the Maven repo on a different directory.

### PR DESCRIPTION
**Pull request by chilanti**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Enterprise Windows users who authenticate via AAD may experience problems with the Java stacks, since those attempt to mount the home directory as the Appsody apps run.

## Related Issues
Fixes #249

### CODE REVIEW

File: content/docs/faq.md [hunk 1]
```diff
@@ -11,16 +11,21 @@ Welcome to the FAQ. Here you will find a list of common questions and answers al
 
 ### 1. Does Appsody support Enterprise Windows users?
 Enterprise Windows users very frequently authenticate through Azure Active Directory (AAD). In those circumstances, the Docker daemon on Windows may not be authorized to access any of the folders that are owned by the AAD user. For this reason, Appsody supports Enterprise Windows users through a workaround.
-1. Create a new project (pick a folder that is outside of your home directory, or Docker will not have access to it).
-2. Go into that project directory and run ```appsody init <stack>```
-3. Create a new folder called ```C:\my-appsody-config-dir\```
-4. Run ```appsody init --config C:\my-appsody-config-dir\.appsody-config.yaml```
+1. Create a new project folder (pick a folder that is outside of your home directory, or Docker will not have access to it).
+1. Create a new folder called ```C:\my-appsody-config-dir\``` (again - any folder outside the home directory would work)
```
Comment (neeraj-laad) [inline content/docs/faq.md]: shall we move steps to create and setup config folder first. and then do all the things in the project folder?

Comment (neeraj-laad) [reply content/docs/faq.md]: Also, did you intend to delete `appsody init <stack>` step?

Comment (chilanti) [reply content/docs/faq.md]: I've addressed those comments. Added back `appsody init <stack>`. I think we need just one call to it, with the appropriate config file...?

File: content/docs/faq.md [hunk 2]
```diff
@@ -11,16 +11,24 @@ Welcome to the FAQ. Here you will find a list of common questions and answers al
...
-4. Run ```appsody init --config C:\my-appsody-config-dir\.appsody-config.yaml```
 
-5. ```notepad C:\my-appsody-config-dir\.appsody-config.yaml```
+1. Create a new folder called ```C:\my-appsody-config-dir\``` (any folder outside the home directory would work)
 
-6. Change the entry ```home:``` to ```C:\my-appsody-config-dir```
+1. Edit the configuration file: ```notepad C:\my-appsody-config-dir\.appsody-config.yaml```
 
-7. Run ```appsody run --config C:\my-appsody-config-dir\.appsody-config.yaml```
+1. Change the entry ```home:``` to ```C:\my-appsody-config-dir```
```
Comment (neeraj-laad) [inline content/docs/faq.md]: I don't think the file would exist at this point? Are we asking users to manually create this file or will this be created by one of the appsody commands? in which case this needs to be in a different order.

Comment (chilanti) [reply content/docs/faq.md]: I've added back the missing step that creates the config file - sorry for the confusion!

Comment (neeraj-laad) [reply content/docs/faq.md]: looks cleaner..

**Pull request merged**